src/name_bazaar/shared/smart_contracts.cljs

```diff
@@ -2,26 +2,33 @@
 
 (def smart-contracts
   {:auction-offering-factory {:name "AuctionOfferingFactory",
-                              :address "0xc806e3343324d1925AD1633b745DD25dbB2BeC75"},
+                              :address "0x959113e1c88d7e866e168ded02bf5cf8aaba2e0f"},
    :buy-now-offering-factory {:name "BuyNowOfferingFactory",
-                              :address "0x23d4a24Fa4981B9b3Ea2F97aA7D87C6055ee08AA"},
-   :name-bazaar-registrar {:name "NameBazaarDevRegistrar",
-                           :address "0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147eA85"},
+                              :address "0xcaf0523f1ca2985b5cbba8a42a8cc413ff7512d0"},
+   ;; Ropsten BaseRegistrarImplementation
+   ;; https://ropsten.etherscan.io/address/0x57f1887a8bf19b14fc0df6fd9b2acc9af147ea85
+   :eth-registrar {:name "BaseRegistrarImplementation",
```

Comment:
Also, the naming should be `NameBazaarDevRegistrar` (and similar for others), because when you redeploy  these names (`:name` keywords) stay the same. You will then have a problem locally when registering domain, because the app will use the `BaseRegistrarImplementation`, which doesn't have the `register` method with single argument arity defined :/

Reply:
I changed it because I didn't like that local-testing-only ABIs are used by default, more so that this config is specifically an example for ropsten deployment, where there is a proper `BaseRegistrarImplementation` contract instead of our patched `NameBazaarDevRegistrar`.

> I don't like how the smart-contracts file works, but it is probably better to leave the names to be NameBazaarDevXYZ - Unless I am missing something :)

Yeah I don't like that either. As it is and since the code is developed more often than deployed (AND using dev contract ABIs wouldn't break otherwise properly configured production deploy anyway), I now agree using the dev configuration is more comfortable too.